docker/Dockerfile.auv-base

```diff
@@ -18,4 +18,6 @@ RUN apt-get update && \
   ros-noetic-xacro \
   ros-noetic-robot-localization && \
   rm -rf /var/lib/apt/lists/*
-
+  rosdep install \
```

Comment:
This needs to be before rm -et /var/lib/apt/lists line


Also after each command do not forget '&& \' instead of '\'

Correct version should be

```
rosdep ... \
  --skip-keys ... \
  --skip-keys ... && \
rm -rf /var/...

```

Reply:
done